### PR TITLE
[test] Port GPUDirect check to Tiger

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix UES-580

Changes needed to run on tiger, is there a better way to do it ?

### CODE REVIEW

File: cscs-checks/mch/gpu_direct_cuda.py [hunk 1]
```diff
@@ -36,4 +36,4 @@ def __init__(self):
                                   self.stdout, 'result', float)
         self.sanity_patterns = sn.assert_reference(result, 1., -1e-5, 1e-5)
         self.maintainers = ['AJ', 'VK']
-        self.tags = {'production', 'mch'}
+        self.tags = {'production', 'mch', 'craype'}
```
Comment (vkarak) [inline cscs-checks/mch/gpu_direct_cuda.py]: @jgphpc Can you do a separate PR adding only the `craype` tag?

**Comment by vkarak** [comment]
@jgphpc To answer your question "is there a better way to do it ?"

I don't think you can  do it differently  for the moment, but in the future we can think of attaching attributes to systems, that we can later  check in the tests. That would decouple the test further.

**Comment by jgphpc** [comment]
https://github.com/eth-cscs/reframe/pull/966#issuecomment-542143143

Would it be possible to detect that we are on a Cray system (`/opt/cray/` or `/etc/opt/cray/release` exist) then we could use stg like:
`if self.current_system.name in {'dom', 'daint', CRAY}:`
instead of
`if self.current_system.name in {'dom', 'daint', 'tiger'}:`
?
